packages/core/src/challenger/isChallengeSubmitTime.ts

```diff
@@ -0,0 +1,18 @@
+import { MINIMUM_TIME_BETWEEN_ASSERTIONS } from "../constants/index.js";
+import { getLatestChallenge } from "./getLatestChallenge.js";
+import { Challenge } from "./index.js";
+
+export async function isChallengeSubmitTime(): Promise<{isSubmitTime:boolean, currentChallenge: Challenge}> {
+
+	    // Get Last Challenge Data
+		const challengeData = await getLatestChallenge();
+		const currentChallenge = challengeData[1];
+		const lastChallengeTime = Number(currentChallenge.assertionTimestamp);
```

Comment:
This is not the correct field, the assertionTimestamp is the time when the Node got created on the Rollup not when the challenge got submitted to the Referee. You need to use `currentChallenge.createdTimestamp`

Reply:
I appreciate the distinction/clarification, this has been corrected!